gulpfile.js

```diff
@@ -105,6 +105,19 @@ const MINIMAL_EXTENSION_SET = [
   'amp-video',
 ];
 
+/**
+ * Extensions to build when `--offline`.
+ * @private @const {!Array<string>}
+ */
+const OFFLINE_EXTENSION_SET = [
```

Comment:
I'd say we just drop this set. Extension sets for the devdash are generated on render time so this makes it hard to keep track. The need for `--extensions` goes away with single-pass anyway.

Reply:
Alternatively, if you really want to add this, you should use the underlying mechanism behind `--extensions-from`.

On startup you just render `/` and get the extension set from that render pass.

Reply:
Ah, so this is what I wanted to do, but I had trouble finding where it grabs the extension set from. And since `/` isn't a static file, will that still work?

Reply:
Discussed offline, split out the `addRequiredExtensionsToHead` into a separate function that just returns the extension set. Have that required in the `gulpfile.js`, and call to get the OFFLINE_EXTENTION_SET when needed 😄